pkg/acceptance/bettertestspoc/assert/objectparametersassert/task_parameters_snowflake_gen.go

```diff
@@ -1027,7 +1027,7 @@ func (t *TaskParametersAssert) HasDefaultTaskAutoRetryAttemptsValueExplicit() *T
 }
 
 func (t *TaskParametersAssert) HasDefaultUserTaskManagedInitialWarehouseSizeValueExplicit() *TaskParametersAssert {
-	return t.HasUserTaskManagedInitialWarehouseSize("Medium")
+	return t.HasUserTaskManagedInitialWarehouseSize(sdk.WarehouseSize("Medium"))
```

Comment:
is this change generated or changed manually?

Reply:
The assert was re-generated, but probably the type has changed in this pr and after re-generation it was still "Medium", so I had to wrap it in `sdk.WarehouseSize` manually.

Reply:
Just checked, the `return t.HasUserTaskManagedInitialWarehouseSize(sdk.WarehouseSize("Medium"))` is from generator